source/_includes/common-tasks/third-party-addons.md

```diff
@@ -1,25 +1,31 @@
-##  Installing third-party add-ons
+## Installing a third-party add-on repository
 
-Home Assistant allows anyone to create add-on repositories to share their add-ons easily. To try this locally, you can use our example add-on repository at
+Home Assistant allows anyone to create an add-on repository to share their own add-ons with the community.
 
-```text
-https://github.com/home-assistant/hassio-addons-example
-```
-
-{% caution %}
+{% warning %}
 Home Assistant cannot guarantee the quality or security of third-party add-ons. Use at your own risk.
-{% endcaution %}
+{% endwarning %}
+
+To add an add-on repository, follow these steps:
+
+1. Copy the URL of the repository.
+   - If you do not have your own repository, but want to try how this works, you can use our example add-on repository:
```

Comment:
This sentence feels a bit strange to me, I'd expect people which get here got linked from a repository or otherwise already know the repository they want to add, and are just interested on where to add it.

I see you use that in the example, so maybe something like:
```suggestion
   - The URL is simply the git repository clone URL (use Code and copy the https/web URL).
   - This documentation uses our example add-on repository, which is not really practically useful but follows the same steps. If you are interested in add-on development, refer to our [add-on development documentation](https://developers.home-assistant.io/docs/add-ons).
```